sw/device/lib/crypto/impl/aes_gcm.c

```diff
@@ -47,23 +47,27 @@ enum {
  *
  * @param internal_ctx Internal context object to save.
  * @param[out] api_ctx Resulting API-facing context object.
+ * @return Result of the operation.
  */
-static inline void gcm_context_save(aes_gcm_context_t *internal_ctx,
-                                    otcrypto_aes_gcm_context_t *api_ctx) {
-  hardened_memcpy(api_ctx->data, (uint32_t *)internal_ctx,
-                  kAesGcmContextNumWords);
+static inline status_t gcm_context_save(aes_gcm_context_t *internal_ctx,
+                                        otcrypto_aes_gcm_context_t *api_ctx) {
+  HARDENED_TRY(hardened_memcpy(api_ctx->data, (uint32_t *)internal_ctx,
+                               kAesGcmContextNumWords));
+  return OTCRYPTO_OK;
 }
 
 /**
  * Restore an AES-GCM context.
  *
  * @param api_ctx API-facing context object to restore from.
  * @param[out] internal_ctx Resulting internal context object.
+ * @return Result of the operation.
  */
-static inline void gcm_context_restore(otcrypto_aes_gcm_context_t *api_ctx,
-                                       aes_gcm_context_t *internal_ctx) {
-  hardened_memcpy((uint32_t *)internal_ctx, api_ctx->data,
-                  kAesGcmContextNumWords);
+static inline status_t gcm_context_restore(otcrypto_aes_gcm_context_t *api_ctx,
+                                           aes_gcm_context_t *internal_ctx) {
+  HARDENED_TRY(hardened_memcpy((uint32_t *)internal_ctx, api_ctx->data,
```

Comment:
same here: hardened try here and hardened try on calling this wrapper function .. any chance to reduce, e.g. here just return hardened_memcpy?